docker/volumeplugin/rancher.go

```diff
@@ -50,7 +50,7 @@ func NewRancherState(driver string, client *client.RancherClient) (*RancherState
 }
 
 func (r *RancherState) IsCreated(name string) (bool, error) {
-	_, vol, err := r.Get(name)
+	_, vol, err := r.GetAny(name)
```

Comment:
I see this is duplicated with the last line: "return isCreated(r.driver, *vol), nil", here isCreated() already check the goodStates[] already.